gate-web/src/test/resources/application-echo.properties

```diff
@@ -34,5 +34,7 @@ services.swabbie.enabled=false
 services.keel.enabled=false
 services.keel.baseUrl=http://nowhere
 
-retrofit.enabled=true
+slack.baseUrl=https://slack.com
+
+retrofit2.enabled=true
```

Comment:
Would we rather depend on having [Retrofit2ServiceFactoryAutoConfiguration](https://github.com/spinnaker/kork/blob/4e1a7974c001b5983d44a24d1b673857fb13f7ac/kork-retrofit2/src/main/java/com/netflix/spinnaker/kork/retrofit/Retrofit2ServiceFactoryAutoConfiguration.java#L32) enabled by default?  Thinking about what happens when it's time to migrate to retrofit3.

Reply:
reverted.